### DISCUSSION TITLE
Does this library support all of MikroTik products that run routeros?

### DISCUSSION THREAD

**Discussion by Nir-Az** [luqasz/librouteros#311] [General]
Does this library support all of MikroTik products that run routeros?
For example: hEX PoE lite (RB750UPr2) or (RB260GSP)?

And can it disable/enable the POE ports using the python API?

**Comment by luqasz** [comment]
>Does this library support all of MikroTik products that run routeros?
For example: hEX PoE lite (RB750UPr2) or (RB260GSP)?

Yes

>And can it disable/enable the POE ports using the python API?

Yes
